tests/test_xdub_calendar.py

```diff
@@ -0,0 +1,161 @@
+from unittest import TestCase
+import pandas as pd
+
+from .test_trading_calendar import ExchangeCalendarTestBase
+from trading_calendars.exchange_calendar_xdub import XDUBExchangeCalendar
+
+
+class XDUBCalendarTestBase(ExchangeCalendarTestBase, TestCase):
+
+    answer_key_filename = 'xdub'
+    calendar_class = XDUBExchangeCalendar
+
+    # The XDUB is open from 8:00 am to 4:28 pm.
+    MAX_SESSION_HOURS = 8.467
+
+    def test_normal_year(self):
+        all_sessions = self.calendar.all_sessions
+
+        expected_holidays = [
+            pd.Timestamp('2018-01-01', tz='UTC'),  # New Year's Day
+            pd.Timestamp('2018-03-30', tz='UTC'),  # Good Friday
+            pd.Timestamp('2018-04-02', tz='UTC'),  # Easter Monday
+            pd.Timestamp('2018-05-07', tz='UTC'),  # May Bank Holiday
+            pd.Timestamp('2018-06-04', tz='UTC'),  # June Bank Holiday
+            pd.Timestamp('2018-08-06', tz='UTC'),  # August Bank Holiday
+            pd.Timestamp('2018-10-29', tz='UTC'),  # October Bank Holiday
+            pd.Timestamp('2018-12-25', tz='UTC'),  # Christmas Day
+            pd.Timestamp('2018-12-26', tz='UTC'),  # Boxing Day
+        ]
+
+        for session_label in expected_holidays:
+            self.assertNotIn(session_label, all_sessions)
+
+        # June Bank Holiday did not start until 2012.
```

Comment:
are we sure this is the case? searching around, I can't find anything specific to the stock exchange, but it looks as though the June bank holiday was observed before that according to https://www.officeholidays.com/countries/ireland/2010.php and some other sources I was looking at. I'm not sure how accurate those are, but it might be good to double check this

Reply:
Thanks for catching this, I'm going to remove this start date.